package.json

```diff
@@ -4,49 +4,42 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
+    "dev:server": "cd server && pnpm run dev",
     "dev": "vite",
-    "build": "vite build",
+    "build": "tsc -b && vite build",
     "lint": "eslint .",
     "preview": "vite preview"
   },
   "dependencies": {
-    "@directus/sdk": "^18.0.3",
-    "@langchain/core": "^0.3.26",
-    "@langchain/openai": "^0.3.16",
     "@radix-ui/react-slot": "^1.1.2",
-    "@tanstack/react-router": "^1.97.0",
+    "@tanstack/react-router": "^1.102.1",
```

Comment:
IMO updating random dependencies that are not part of the scope of the PR is not good.

If Shurti is fine with fine that's fine

Reply:
+1